test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -291,49 +292,50 @@ describe('global navigation', () => {
   });
 
   describe('Viewport changes', () => {
-    it('should render desktop -> small desktop -> mobile', async () => {
```

Comment:
Are this suppose to be still commented?

Reply:
Added back commented test cases for older GNAV UT's.